### PR TITLE
Fix parameter sorting

### PR DESCRIPTION
**Pull request by nils-braun**
Superseeds #641 and adds tests.
Thanks to @trueckel for the implementation.

### CODE REVIEW

**Comment by nils-braun** [comment]
@MaxBenChrist what do you think? The feature @trueckel implemented is actually very reasonable, but it breaks our API (and maybe the code of many people).
Need your OK here :-)

**Comment by MaxBenChrist** [comment]
> @MaxBenChrist what do you think? The feature @trueckel implemented is actually very reasonable, but it breaks our API (and maybe the code of many people).
> Need your OK here :-)

I am not seeing why this breaks our API?

File: tsfresh/feature_extraction/feature_calculators.py [hunk 1]
```diff
@@ -12,6 +12,9 @@
 They are specified using the "fctype" parameter of each feature calculator, which is added using the
 set_property function. Only functions in this python module, which have a parameter called  "fctype" are
 seen by tsfresh as a feature calculator. Others will not be calculated.
+
+Feature calculators of type combiner should return the concatenated parameters sorted
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py:16]: This is garantueed by https://github.com/blue-yonder/tsfresh/blame/54de174f593290a0c798ccad2ad7cad493c9628e/tsfresh/utilities/string_manipulation.py#L70, isn't?

Comment (nils-braun) [reply tsfresh/feature_extraction/feature_calculators.py:16]: Well unfortunately no. Only if it is a non-combiner...

**Comment by nils-braun** [comment]
@MaxBenChrist I will merge it nevertheless. I does break our API if people really use the extracted feature names in the code. We would need to make sure to communicate this clearly in the net version I guess?

**Pull request merged**